app.json

```diff
@@ -68,6 +68,26 @@
             "min": -100,
             "max": 0,
             "step": 1
+        },
+        "measure_bytes_send":{
```

Comment:
I think this should be named measure_bytes_sent (past tense), to make it in correct english grammar.